Tests/js/views/tabs/assets/ez-locationviewdetailstabview-tests.js

```diff
@@ -0,0 +1,287 @@
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Render should call the template": function () {
+            var templateCalled = false,
+                origTpl;
+
+            origTpl = this.view.template;
+            this.view.template = function () {
+                templateCalled = true;
+                return origTpl.apply(this, arguments);
+            };
+            this.view.render();
+            Y.Assert.isTrue(templateCalled, "The template should have been used to render this.view");
+        },
+    });
+
+    changeEventTest = new Y.Test.Case({
+        name: "eZ LocationViewDetailsTabView change event test",
+        setUp: function () {
+            this.view = new Y.eZ.LocationViewDetailsTabView({});
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        _authorsEventTest: function (eventName) {
+            var renderCalled = false;
+
+            this.view.render = function () {
+                renderCalled = true;
+            };
+
+            this.view.fire(eventName);
+
+            Assert.isTrue(
+                renderCalled,
+                "Render should have been called by the event: " + eventName
+            );
+        },
+
+        "Test that creator change event calls render": function () {
+            this._authorsEventTest('creatorChange');
+        },
+
+        "Test that owner change event calls render": function () {
+            this._authorsEventTest('ownerChange');
+        },
+    });
+
+    fireLoadUserEventTest = new Y.Test.Case({
+        name: "eZ LocationViewDetailsTabView fire load user event test",
+        setUp: function () {
+            var that = this;
+
+            this.contentMock = new Mock();
+            this.versionMock = new Mock();
+            this.creator = "13";
+            this.owner = "42";
+
+            Mock.expect(that.contentMock, {
+                method: 'get',
+                args: [Mock.Value.String],
+                run: function (attr) {
+                    if (attr === 'currentVersion') {
+                        return that.versionMock;
+                    } else if (attr === 'resources') {
+                        return {Owner: that.owner};
+                    } else {
+                        Y.fail("Unexpected parameter " + attr + " for content mock");
+                    }
+                }
+            });
+
+            Mock.expect(that.versionMock, {
+                method: 'get',
+                args: ['resources'],
+                returns: {Creator: this.creator}
+            });
+
+            this.view = new Y.eZ.LocationViewDetailsTabView({
+                content: that.contentMock
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Should fire loadUser Event with different owner and creator": function () {
```

Comment:
actually you are testing that the `loadUser` event is fired twice so title should be named:
*Should fire the loadUser event twice with a different owner and creator*